specifyweb/frontend/js_src/lib/components/QueryBuilder/Wrapped.tsx

```diff
@@ -247,6 +247,14 @@ export function QueryBuilder({
   );
   const resultsShown = state.queryRunCount !== 0;
 
+  const [collapseAllPref] = userPreferences.use(
+    'queryBuilder',
+    'behavior',
+    'collapseQueryLine'
+  );
+  const [isAllCollapsed, setIsAllCollapsed] =
+    React.useState<boolean>(collapseAllPref);
```

Comment:
(optional) Consider getting rid of React.useState and using only `userPreferences.use`
Benefits:
1. the value will be preserved between sessions (consider if this is desirable behavior in this case)
2. you won't have to pass the state down to other components - other components can just call the `userPreferences.use` once again (any changed to the same `userPreferences.use` in one component is synced with all other components)